index.html

```diff
@@ -1,54 +1,36 @@
-<html lang="en">
-  <head>
-    <title>Guess the Number</title>
-    <link
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      rel="stylesheet"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
-    />
-    <link rel="stylesheet" type="text/css" href="style.css" />
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Guess The Number</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body class="text-center">
+</head>
+<body class="text-center">
+
     <nav class="navbar navbar-dark bg-dark">
-      <!-- Navigation bar content -->
-      <a class="navbar-brand" href="#">Guess The Number</a>
+        <!-- Navbar content -->
+        <a class="navbar-brand" href="#"><h1>Guess The Number</h1></a>
     </nav>
-
-    <main>
-      <h1 class="banner p-3">1...100</h1>
-
-      <div class="form-group">
-        <input
-          id="number-guess"
-          class="form-control form-control-lg"
-          type="number"
-          placeholder="What's your guess?"
-        />
-      </div>
-
-      <div id="result">
-        <!-- This is where the result will be -->
-      </div>
-
-      <form>
-        <button
-          type="button"
-          id="number-submit"
-          class="btn btn-lg btn-secondary"
-        >
-          Check Me
-        </button>
-
-        <button type="button" id="restart-game" class="btn btn-lg btn-light">
-          Restart
-        </button>
-      </form>
-
-      <div id="history">
-        <!-- This is where the history will be -->
-      </div>
-    </main>
-  </body>
-</html>
+    <div class="row">
+        
+        <div class="side-title card">
+            <h1>Guess a number from 1 to 100</h1>
+            <div class="input-row" id="remove-input">
```

Comment:
- [ ] Change the title Guess a number from 1 to 100. Use a h2 tag and say 1>2>>>100 or 1--2---100

- [ ] You have to take the history out of the input div. When the history increases the input div also increases. Input div has to be fixed size.

- [ ] Add a heading called History while showing history.

- [ ] Add a stylish image border around history.

Reply:
Can you check if the heading is h2 or can you make the heading on the first task a bit bigger?